utils/math.js

```diff
@@ -0,0 +1,25 @@
+// math.js
+
+// Function to generate a random integer within a specified range
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
```

Comment:
@gblotta-xoor There is a problem here:
- We are redefining the values for some parameters (`min` and `max`).

- 
```suggestion
function getRandomInt(min, max) {
  const transformedMin = Math.ceil(min);
  const transformedMax = Math.floor(max);
  return Math.floor(Math.random() * (transformedMax - transformedMin + 1)) + transformedMin;
}
```

Reply:
@jaymdq You're right here 👍 https://github.com/jaymdq/code-review-process-workshop/pull/3/commits/c6500744205e7fcb99367260cda4d5e02993aee4